src/DebuggerActions/ReturnValueDebugAction.class.st

```diff
@@ -39,6 +39,11 @@ ReturnValueDebugAction >> executeAction [
 	self session returnValue: value from: self currentContext
 ]
 
+{ #category : #accessing }
+ReturnValueDebugAction >> help [
+	^ 'Forces a return of a given value to the previous context.'
```

Comment:
"Force return..." or just "Return..." maybe? Just to be consistent in the form of versbs.